api/server/testutils_test.go

```diff
@@ -52,6 +55,11 @@ func newTestCluster(t *testing.T) *testCluster {
 	// Create a new mock cluster
 	tester.c = mockcluster.NewMockCluster(tester.mc)
 
+	// Create a new mock Serets
+	// This doesn't set *mocksecrets.Mocksecrets, thats why needed to
+	// write newTestClusterSecrets() and newTestServerSecrets()
+	//tester.sm = mocksecrets.NewMockSecrets(tester.mc)
```

Comment:
@lpabon ,  is there any way to achieve this, thus avoid writing different  newTestClusterSecrets() and newTestServerSecrets()?

Reply:
@ram-infrac Hi, I'm not sure what you mean. Could you explain the pros and cons or the issue? What is the difference between newTestServerSecret and newTestClusterSecret? Is newTestServerSecret needed?